src/main/java/org/spongepowered/api/effect/potion/PotionEffect.java

```diff
@@ -57,11 +57,12 @@ static Builder builder() {
      * duration in ticks.
      *
      * @param type The potion type
-     * @param amplifier The amplifier
+     * @param amplifier The zero-indexed amplifier
```

Comment:
While this is true, I don't think this itself is particularly helpful. It's zero indexed, so what does that mean for the amplifier itself? It's really an implementation note, not a descriptive note.

Would we be better served either:

* making this one-indexed and handling the change to zero based ourselves, or
* being a bit more descriptive about what the amplifier is, e.g the first level of a potion is amplifier zero, which each potion level having a higher amplifier (level-1, to be precise).

Reply:
At the very least it should definitely be more descriptive, I'll get to that in a little bit. It would probably be sensible to make it one-indexed though it does sound like a kinda unnecessary breaking change when some better javadoc wording would probably suffice

Reply:
I agree with you there - just needs a bit more of a descriptive wording. I'll pull this in as is and quickly update it as I'm trying to get around tying up the API today, but thank you for the updates as is!